pkg/plugin/extension_builder_test.go

```diff
@@ -75,6 +84,7 @@ func TestExtensionBuilder_Build(t *testing.T) {
 			description:   i18n.StringFrom("ddd"),
 			schema:        id.MustPropertySchemaID("foo#1.1.1/hhh"),
 			visualizer:    "vvv",
+			widgetLayout:  &scene.WidgetLayout{},
```

Comment:
As I mentioned in another comment, don't empty it, fill it with something inside.